refactor: fix nonexisting feature

b-zee opened the pull request:
The `wasm32` is not a feature. It was probably supposed to be the target architecture.

davidrusu commented:
bors try

dirvine reviewed (approved):
Perfect

dirvine reviewed (approved):
Perfect!! we need this in for next testnet

b-zee committed:
refactor: fix nonexisting feature

The `wasm32` is not a feature. It was probably supposed to be the target architecture.
